hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ConfigCheck.java

```diff
@@ -208,7 +202,7 @@ public void writeData(ObjectDataOutput out) throws IOException {
     @Override
     public void readData(ObjectDataInput in) throws IOException {
```

Comment:
As mentioned [here](https://hazelcast.atlassian.net/wiki/display/EN/Prevent+Group+Password+Broadcast+Design?focusedCommentId=130520070#comment-130520070), the alternative to doing this in 3.8.x is making the change in 3.9 only and send the password if the cluster version is lower than 3.9:
```
if (out.getVersion().isLessOrEqual(Versions.V3_8)){
            out.writeUTF(groupPassword);
        }
```
```
if (in.getVersion().isLessOrEqual(Versions.V3_8)){
            in.readUTF();
        }
```